tests/unit/components/boolean-input-test.js

```diff
@@ -44,6 +44,30 @@ describeComponent(
       ])
     })
 
+    it('shows error messages when store.showErrorMessages and showErrorMessages are true', function () {
+      component.set('showErrorMessages', false)
+      component.set('store.showAllErrors', true)
+      expect(component.get('showErrorMessages')).to.be.truthy
+    })
```

Comment:
Two issues here:
 1. The text in the `it()` doesn't match what's being tested, it says both are true, but `showErrorMessages` is starting out `false`
 1. Shouldn't we be checking something like `component.get('renderErrorMessage')` vs. `component.get('showErrorMessage')`? With the test as-is, I'm fairly certain you're not exercising anything to do with `store.showAllErrors`, you're just setting `x` to be `true` then expecting `x` to be `true` still :)